index.js

```diff
@@ -77,6 +78,21 @@ function send(ctx, path, opts) {
       ctx.res.removeHeader('Content-Length');
     }
 
+    if (extensions && !/\..*$/.exec(path)) {
+      var list = [].concat(extensions || []);
+      for (var i = 0; i < list.length; i++) {
+        var extension = list[i];
+        if (!(typeof extension === 'string' || extension instanceof String)) {
```

Comment:
`if (typeof extension !== 'string')` is enough ((^_^)).

@Necromos I'll merge (rebase) this PR and publish a new version before this weekend.

Great work ! 👍